ToDoApp/ToDoApp/Assets.xcassets/primaryText.colorset/Contents.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
It is good practice to have colours named in the same convention. Either [camel case](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwi2p_7_wOP7AhXSPn0KHRGxDEAQFnoECBEQAQ&url=https%3A%2F%2Fen.wikipedia.org%2Fwiki%2FCamel_case&usg=AOvVaw3rpk9xbQSt-mdoJTG9ynM5) or [pascal case](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwj185uHweP7AhUSIjQIHTgoAvEQFnoECAoQAQ&url=https%3A%2F%2Fwww.theserverside.com%2Fdefinition%2FPascal-case&usg=AOvVaw14ft2V8YkEISz_Z-zF0pLN). I would suggest using the camel case convention. Will show you later why

Reply:
I think I have renamed the colour name to camel case: primaryText, primaryButton, etc. Is it as your expectation?

Reply:
@hieuvuong2310 yes 👍🏻